tools/tools.go

```diff
@@ -6,7 +6,6 @@ import (
 	// document generation
 	_ "github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs"
 	// SDK upgrade
```

Comment:
I presume this line should be removed also @trentrosenbaum

Reply:
For a moment I misread this one and focused on the import.  I have removed the SDK comment.